benchmarks/benchmark/tools/locust-load-inference/README.md

```diff
@@ -8,10 +8,12 @@
     - [Step 2: create and give service account access to view dataset](#step-2-create-and-give-service-account-access-to-view-dataset)
     - [Step 3: create output bucket](#step-3-create-output-bucket)
     - [Step 4: create and give service account access to write to output gcs bucket](#step-4-create-and-give-service-account-access-to-write-to-output-gcs-bucket)
+    - [Step 6: create and configure terraform.tfvars](#step-6-create-and-configure-terraformtfvars)
```

Comment:
Can you change the numbering? It stays step 6.

Reply:
ty for pointing this out, fixed!